src/content/en/2020/third-parties.md

```diff
@@ -8,16 +8,261 @@ authors: [simonhearne]
+
+Third-party resources are:
+
+* Hosted on a _shared_ and _public_ origin
+* Widely used by a variety of sites
+* Uninfluenced by an individual site owner
+
+To match these goals as closely as possible, the formal definition used throughout this chapter for third-party resources is: A resource that originates from a domain whose resources can be found on at least 50 unique pages in the HTTP Archive dataset.
+
+Note that using these definitions, third-party content served from a first-party domain is counted as a first-party content. For example: Self-hosting Google Fonts or bootstrap.css is counted as _first-party content_.  Similarly, first-party content served from a third-party domain is counted as third-party content. An associated example: First-party images served over a CDN on a third-party domain are considered _third-party content_.
+
+### Provider categories
+
+This chapter divides third-party providers into different categories. A brief description is included with each of the categories.  The mapping of domain to category can be found in the [third-party-web repository](https://github.com/patrickhulce/third-party-web/blob/8afa2d8cadddec8f0db39e7d715c07e85fb0f8ec/data/entities.json5).
+
+* Ad - display and measurement of advertisements
+* Analytics - tracking site visitor behavior
+* CDN - providers that host public shared utilities or private content of their users
+* Content - providers that facilitate publishers and host syndicated content
+* Customer Success - support and customer relationship management functionality
+* Hosting - providers that host the arbitrary content of their users
+* Marketing - sales, lead generation, and email marketing functionality
+* Social - social networks and their affiliated integrations
+* Tag Manager - provider whose sole role is to manage the inclusion of other third parties
+* Utility - code that aids the development objectives of the site owner
+* Video - providers that host the arbitrary video content of their users
+* Other - uncategorized or non-conforming activity
+
+_Note on CDNs: The CDN category here includes providers that provide resources on public CDN domains (e.g. bootstrapcdn.com, cdnjs.cloudflare.com, etc.) and does not include resources that are simply served over a CDN. i.e. putting Cloudflare in front of a page would not influence its first-party designation according to our criteria._
+
+### Caveats
+
+* All data presented here is based on a non-interactive, cold load. These values could start to look quite different after user interaction.
+* The pages are tested with no cookies set, so third-parties requested after opt-in are not included. This will especially affect pages hosted and predominantly served to countries in scope for the [General Data Protection Regulation](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation).
```

Comment:
In the context of GDPR, it may also be important to mention that all of these measurements were performed on servers in the US.  

Also worth including a link to the methodolgy - https://almanac.httparchive.org/en/2020/methodology